src/Core/Grid/Definition/Factory/CmsPageDefinitionFactory.php

```diff
@@ -166,6 +168,16 @@ protected function getColumns()
             ->setName($this->trans('Actions', [], 'Admin.Global'))
             ->setOptions([
                 'actions' => (new RowActionCollection())
+                    ->add((new LinkRowAction('view'))
+                    ->setName($this->trans('View', [], 'Admin.Actions'))
+                    ->setIcon('visibility')
+                    ->setOptions([
+                        'route' => 'admin_cms_pages_view',
+                        'route_param_name' => 'cmsPageId',
+                        'route_param_field' => 'id_cms',
+                        'use_inline_display' => true,
```

Comment:
Seems like this action opens a preview of the CMS page, then maybe it should be opened in another tab
```suggestion
                        'use_inline_display' => true,
                        'target' => '_blank',
```